recipe/meta.yaml

```
@@ -86,13 +81,12 @@ test:
     - pytest >=7.1.2
     - pytest-timeout
     - pytest-xdist
-    - matplotlib-base >=3.5.0
+    - matplotlib-base >=3.6.1
     - scikit-image >=0.19
     - pandas >=1.4.0
-    # latest polars in main = 0.20.18
-    #- polars >=0.20.23
-    # pyarrow is not available for python 3.14 on the main channel yet
-    - pyarrow >=12.0.0  # [py<314]
+    # Latest polars available for py313
+    - polars >=0.20.30  # [py<314]
+    - pyarrow >=12.0.0
     # tests docstring failed with numpydoc >=1.9.0
     - numpydoc >=1.2.0,<1.9.0  # [py<313]
     - pooch >=1.6.0
```

https://github.com/scikit-learn/scikit-learn/blob/1.8.0/pyproject.toml#L90


```suggestion
    - pooch >=1.8.0
```